modules/web-console/web-agent/src/test/java/org/apache/ignite/testsuites/IgniteWebAgentTestSuite.java

```diff
@@ -30,6 +31,7 @@
 @Suite.SuiteClasses({
     AgentLauncherTest.class,
     AgentUtilsTest.class,
+    WebSocketRouterTest.class,
```

Comment:
Lets sort tests alphabetically?

Reply:
I sorted by import order. It's more easy to check if you look at project structure

Reply:
No, imports usually folded. And You could navigate in IDE by ctrl+click on class name.

Reply:
I use "project" window in IDE to validate that all tests was added to suite.

Reply:
OK